doc/classes/Redirect.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="Redirect" inherits="Object" version="4.0">
+	<brief_description>
+	</brief_description>
+	<description>
+	</description>
+	<tutorials>
+	</tutorials>
+	<signals>
+		<signal name="print_line">
+			<argument index="0" name="scene_root" type="Node" />
+			<description>
```

Comment:
Please will in the docs/description for the new signal here in `<description></description>` :)

Reply:
yessir, on it, will do it in a sec.

Reply:
Nevermind, after recompiling I have the text in the editor :-)